### PR TITLE
added alert and button

### PR DESCRIPTION
**Pull request by bishnoimukesh**
# Change Logs

## Screenshot
alert-and-button

## Deploy Preview Link
https://deploy-preview-2--compi-ui.netlify.app/

### CODE REVIEW

File: components/alert/alert.html [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+    <nav class="navigation">
+        <div class="nav-main-heading">Getting Started</div>
+
+        <div class="nav-sub-heading nav-getstarted">
+            <a href="../home.html">About</a>
+        </div>
+
+        <div class="nav-main-heading">Components</div>
+        <ul class="nav-comp-section">
```
Comment (xii-vi) [inline components/alert/alert.html:44]: if you are using short naming convention, use it everywhere

**Review by xii-vi — APPROVED** [review]
Some changes are suggested. Rest is fine to merge.

**Review by sharmayatendra — COMMENTED** [review]
everything looks fine.

**Comment by bishnoimukesh** [comment]
all the necessary changes are made

**Pull request merged**